markhor_flippers/src/markhor_hw_interface_flippers.cpp

```diff
@@ -590,7 +617,7 @@ void MarkhorHWInterfaceFlippers::applyDrivePosition(std::unique_ptr<TalonSRX>& d
   }
   do
   {
-    error = drive->GetSensorCollection().SetPulseWidthPosition(drive_position, timeout_ms_);
+    error = drive->SetSelectedSensorPosition(-1 *drive_position, 0,timeout_ms_);
```

Comment:
Nitpick: missing spaces between the arguments. Please use the clang-format tool, please.